fontir/src/ir.rs

```diff
@@ -193,29 +193,39 @@ impl IntoIterator for GlyphOrder {
 pub type PostscriptNames = HashMap<GlyphName, GlyphName>;
 
 /// In logical (reading) order
-type KernPair = (KernParticipant, KernParticipant);
-type KernValues = BTreeMap<NormalizedLocation, OrderedFloat<f32>>;
+pub type KernPair = (KernParticipant, KernParticipant);
 
-/// IR representation of kerning.
+/// IR representation of kerning groups.
 ///
 /// In UFO terms, roughly [groups.plist](https://unifiedfontobject.org/versions/ufo3/groups.plist/)
-/// and [kerning.plist](https://unifiedfontobject.org/versions/ufo3/kerning.plist/) combined.
+/// plus the set of locations that can have kerning.
 #[derive(Serialize, Deserialize, Debug, Default, Clone, PartialEq, Eq)]
-pub struct Kerning {
+pub struct KerningGroups {
     pub groups: BTreeMap<GroupName, BTreeSet<GlyphName>>,
+    /// The locations that have kerning defined.
+    ///
+    /// Must be a subset of the master locations.
+    pub locations: BTreeSet<NormalizedLocation>,
```

Comment:
now that i'm actually working in this file again, what's the rationale for having the locations here? Is it related to groups in particular in any way?

Reply:
Those are the locations you should spawn kern instance jobs for